doc/conf.py

```diff
@@ -308,29 +308,93 @@ def _check_dependencies():
 latex_logo = None
 
 latex_elements = {}
+
+latex_engine = 'xelatex'  # many Unicode glyphs in our docs...
+# By default Sphinx with latex_engine = 'xelatex' uses polyglossia
+# If this key is changed, clean the build/latex directory before next build
+latex_elements['babel'] = r'\usepackage{babel}'
+
 # Additional stuff for the LaTeX preamble.
+
+# one line per author name on title page
 latex_elements['preamble'] = r"""
-   % In the parameters section, place a newline after the Parameters
-   % header.  (This is stolen directly from Numpy's conf.py, since it
-   % affects Numpy-style docstrings).
-   \usepackage{expdlist}
-   \let\latexdescription=\description
-   \def\description{\latexdescription{}{} \breaklabel}
-
-   \usepackage{amsmath}
-   \usepackage{amsfonts}
-   \usepackage{amssymb}
-   \usepackage{txfonts}
-
-   % The enumitem package provides unlimited nesting of lists and
-   % enums.  Sphinx may use this in the future, in which case this can
-   % be removed.  See
-   % https://bitbucket.org/birkenfeld/sphinx/issue/777/latex-output-too-deeply-nested
-   \usepackage{enumitem}
-   \setlistdepth{2048}
+  \DeclareRobustCommand{\and}{%
+    \end{tabular}\kern-\tabcolsep\\\begin{tabular}[t]{c}%
+  }%
 """
+
+# Usage of expdlist package (as originally copied over from Numpy's confpy)
```

Comment:
It in ok to just remove this, the comment in the commit log is good enough.

Reply:
As this commit properly belongs to #15797, I will do the change over there and keep here only the fix proper for author display on title page.